src/mock/person/personMock.ts

```diff
@@ -41,6 +75,9 @@ function getTilfeldigPerson(fødselsnummer: string): Person {
         statsborgerskap: getStatsborgerskap(),
         personstatus: getPersonstatus(alder),
         bankkonto: getBankKonto(),
+        folkeregistrertAdresse: getTilfeldigAdresse(),
```

Comment:
For å unngå at denne filen blir alt for stor og oversiktelig kan du vurdere å flytte litt adresseMocking ut i en egen fil :)